src/framework/preferences/js/Panels.js

```diff
@@ -692,6 +692,93 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     });
 
+    /********************************************************
+     * A base grade for styled radio Button adjuster panels *
+     ********************************************************/
+
+    fluid.defaults("fluid.prefs.panel.styledRadioButtonAdjuster", {
+        gradeNames: ["fluid.prefs.panel"],
+        mergePolicy: {
+            "controlValues.theme": "replace",
+            "stringArrayIndex.theme": "replace"
+        },
+        listeners: {
+            "afterRender.style": "{that}.style"
+        },
+        selectors: {
+            themeRow: ".flc-prefsEditor-themeRow",
+            themeLabel: ".flc-prefsEditor-theme-label",
+            themeInput: ".flc-prefsEditor-themeInput",
+            label: ".flc-prefsEditor-styledRadioButton-label",
+            description: ".flc-prefsEditor-styledRadioButton-descr"
+        },
+        styles: {
+            defaultThemeLabel: "fl-prefsEditor-styledRadioButton-defaultThemeLabel"
+        },
+        stringArrayIndex: {
```

Comment:
Probably move blocks of `stringArrayIndex` and `controlValues` to the top within component options as they are expected to be provided by integrators. It might also be helpful to add code comments stating: 1. they are expected to be supplied; 2. their purposes.